README.md

```diff
@@ -128,6 +128,9 @@ tests/scripts/all_tests.sh
 
 PyPi releases are automatically pushed when creating a new Git tag/Github release. 
 
+## API documentation
+Check out the [latest version of the API documentation](https://static.webknossos.org/lib-docs/master/wkcuber/api.html).
```

Comment:
:+1: just wanted to suggest that

Reply:
Wrong link? (at least does not work for me) Did you mean `https://static.webknossos.org/lib-docs/docs_s3/wkcuber/api.html`?

Reply:
The link will work, once it is deployed on `master` branch

Reply:
Never mind